src/components/manage/Preferences/PersonalInformation.jsx

```diff
@@ -162,70 +100,21 @@ class PersonalInformation extends Component {
    */
   render() {
     return (
-      <Form
-        formData={this.props.user}
-        schema={{
-          fieldsets: [
-            {
-              id: 'default',
-              title: this.props.intl.formatMessage(messages.default),
-              fields: [
-                'fullname',
-                'email',
-                'portrait',
-                'home_page',
-                'location',
-              ],
-            },
-          ],
-          properties: {
-            fullname: {
-              description: this.props.intl.formatMessage(
-                messages.fullnameDescription,
-              ),
-              title: this.props.intl.formatMessage(messages.fullnameTitle),
-              type: 'string',
-            },
-            email: {
-              description: this.props.intl.formatMessage(
-                messages.emailDescription,
-              ),
-              title: this.props.intl.formatMessage(messages.emailTitle),
-              type: 'string',
-            },
-            portrait: {
-              description: this.props.intl.formatMessage(
-                messages.portraitDescription,
-              ),
-              title: this.props.intl.formatMessage(messages.portraitTitle),
-              type: 'object',
-            },
-            home_page: {
-              description: this.props.intl.formatMessage(
-                messages.homePageDescription,
-              ),
-              title: this.props.intl.formatMessage(messages.homePageTitle),
-              type: 'string',
-            },
-            location: {
-              description: this.props.intl.formatMessage(
-                messages.locationDescription,
-              ),
-              title: this.props.intl.formatMessage(messages.locationTitle),
-              type: 'string',
-            },
-          },
-          required: ['email'],
-        }}
-        onSubmit={this.onSubmit}
-        onCancel={this.onCancel}
-        loading={this.props.loading}
-      />
+      this.props?.userschema?.loaded && (
+        <Form
```

Comment:
Tested and looks fine functionality wise. We need to make a decision on having two profile forms( schema-based and default) and the intended route placing. Currently the schema form lives in `/personal-information` which seems fine.  

To be discussed in next meeting.